src/expressionDescriptor.ts

```diff
@@ -14,7 +14,16 @@ export class ExpressionDescriptor {
   expressionParts: string[];
   options: Options;
   i18n: Locale;
-
+  parseDetails: {
+    full?: string | null;
+    min?: string | null;
+    hour?: string | null;
```

Comment:
Can we expand this so it's more descriptive - maybe `hours`?

Reply:
Yes. it is up to you